survey_form/index.html

```diff
@@ -43,8 +43,94 @@
 
     <div class="tittle"><h1 class="tittle__item">Survey Form</h1></div>
 
-    <form class="form">
-      <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+    <form class="form-survey">
+      <legend class="form-survey__title">
+        Lets us now how we can improve freeCodeCamp
+      </legend>
+
+      <label class="form-survey__label" for="name">Name: </label>
+      <input
+        type="text"
+        id="name"
+        class="form-survey__input"
+        required
+        placeholder="Leonardo DiCaprio"
+      />
+
+      <label class="form-survey__label" for="email">Email: </label>
+      <input
+        type="email"
+        id="email"
+        class="form-survey__input"
+        required
+        placeholder="leonardo.dicaprio@gmail.com"
+      />
+
+      <label class="form-survey__label" for="age">Age: </label>
+      <input
+        type="age"
+        id="number"
+        class="form-survey__input"
+        placeholder="44"
+        min="12"
+        max="123"
+        required
+      />
+
+      <label class="form-survey__label" for="current_role"
+        >Which option best describes your current role?</label
+      >
+
+      <select class="form-survey__dropdown" id="current_role">
+        <option value="select_an_option" disabled>Select an option</option>
+        <option value="student">Student</option>
+        <option value="full_time_job">Full time job</option>
+        <option value="full_time_learner">Full time learner</option>
+        <option value="prefer_not_to_say">Prefer not to say</option>
+        <option value="other">Other</option>
+      </select>
+
+      <label
+        class="form-survey__label form-survey__label--radiobutton"
+        for="form-survey__radiobutton"
+        >How likely is that you would recommend freeCodeCamp to a friend?</label
+      >
+
+      <div class="form-survey__radiobutton" id="form-survey__radiobutton">
+        <div class="form-survey__radiobutton--group">
+          <input
+            class="form-survet__input--radiobutton"
+            type="radio"
+            name="definitely"
+            id="definitely"
+          />
+          <label class="form-survey__input--label" for="definitely"
+            >Definitely</label
+          >
+        </div>
+
+        <div class="form-survey__radiobutton--group">
+          <input
+            class="form-survet__input--radiobutton"
+            type="radio"
+            name="maybe"
+            id="maybe"
+          />
+          <label class="form-survey__input--label" for="maybe">Maybe</label>
+        </div>
+
+        <div class="form-survey__radiobutton--group">
+          <input
+            class="form-survet__input--radiobutton"
+            type="radio"
+            name="notsure"
+            id="not_sure"
+          />
+          <label class="form-survey__input--label" id="not_sure"
```

Comment:
Zamiast `id` powinno być `for`. Możesz to przetestować tak, że po kliknięciu na tekst labela, powinien zaznaczać sie input mu odpowiadający.